src/app/notifications/components/SearchFailure/styles.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import styled, { css, keyframes } from 'styled-components/macro';
+import { PlainButton } from '../../../components/Button';
+import Times from '../../../components/Times';
+import {
+  bookBannerDesktopMiniHeight,
+  bookBannerMobileMiniHeight,
+  toolbarDesktopHeight,
+  toolbarMobileHeight,
+  toolbarMobileSearchWrapperHeight
+} from '../../../content/components/constants';
+import { disablePrint } from '../../../content/components/utils/disablePrint';
+import theme from '../../../theme';
+import { inlineDisplayBreak } from '../../theme';
+import { Header } from '../Card';
+
+export const clearErrorAfter = 3200;
+const fadeOutDuration = 1000;
+
+const bannerBackground = '#F8E8EB';
```

Comment:
unless these colors are totally made up randomly they should be pulled out of app/theme.colors

Reply:
Yes, the colors were previously in theme, now just constants in this file